test/apns_test.exs

```diff
@@ -419,4 +419,37 @@ defmodule Sparrow.APNSTest do
     assert auth.certfile == path_to_cert
     assert auth.keyfile == path_to_key
   end
+
+  test "APNS empty alert config" do
+    with_mock Sparrow.H2Worker.Pool,
+      send_request: fn _, r, _, _, _ ->
+        headers = [{":status", "200"} | r.headers]
+        send(self(), {:ok, {headers, r.body}})
+        {:ok, {headers, r.body}}
+      end do
+      notification =
+        "EchoBodyHandler"
+        |> Notification.new(:dev)
+        |> Notification.add_custom_data("gameID", "192837")
+        |> Notification.add_category("GAME_INVITATION")
+
+      :ok = Sparrow.APNS.push(@pool_name, notification)
+
+      {:ok, {_headers, body}} =
+        receive do
+          {:ok, {headers, body}} -> {:ok, {headers, body}}
+        after
+          1_000 -> assert false
+        end
+
+      {:ok, response} = Jason.decode(body)
+      expected_response =
+        "{\"aps\":
```

Comment:
How about extracting it to module parameter?

Reply:
If you can generate, that response by sending real request it would be great to use that ;)